### PR TITLE
Release 15.0.0

### PR DESCRIPTION
**Pull request by FrederikBolding**
See the changelogs for details.

### CODE REVIEW

File: packages/snaps-controllers/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.2.0]
+### Added
+- Add support for links in Custom UI and Notifications ([#1814](https://github.com/MetaMask/snaps/pull/1814))
+
+### Fixed
+- Fix an issue with throwing snaps running long ([#1897](https://github.com/MetaMask/snaps/pull/1897))
```
Comment (Mrtenz) [inline packages/snaps-controllers/CHANGELOG.md:14]: This sentence is confusing

File: packages/snaps-controllers/CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.2.0]
+### Added
+- Add support for links in Custom UI and Notifications ([#1814](https://github.com/MetaMask/snaps/pull/1814))
```
Comment (Mrtenz) [inline packages/snaps-controllers/CHANGELOG.md:11]:
```suggestion
- Add support for links in custom UI and notifications ([#1814](https://github.com/MetaMask/snaps/pull/1814))
```

File: packages/snaps-jest/CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.1.0]
+### Added
+- Add support for links in Custom UI and Notifications ([#1814](https://github.com/MetaMask/snaps/pull/1814))
```
Comment (Mrtenz) [inline packages/snaps-jest/CHANGELOG.md:11]:
```suggestion
- Add support for links in custom UI and notifications ([#1814](https://github.com/MetaMask/snaps/pull/1814))
```

File: packages/snaps-rpc-methods/CHANGELOG.md [hunk 4]
```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.2.0]
+### Added
+- Add support for links in Custom UI and Notifications ([#1814](https://github.com/MetaMask/snaps/pull/1814))
```
Comment (Mrtenz) [inline packages/snaps-rpc-methods/CHANGELOG.md:11]:
```suggestion
- Add support for links in custom UI and notifications ([#1814](https://github.com/MetaMask/snaps/pull/1814))
```

File: packages/snaps-simulator/CHANGELOG.md [hunk 5]
```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.2.0]
+### Added
+- Add support for links in Custom UI ([#1814](https://github.com/MetaMask/snaps/pull/1814))
```
Comment (Mrtenz) [inline packages/snaps-simulator/CHANGELOG.md:11]:
```suggestion
- Add support for links in custom UI ([#1814](https://github.com/MetaMask/snaps/pull/1814))
```

File: packages/snaps-ui/CHANGELOG.md [hunk 6]
```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.1.0]
+### Added
+- Add support for links in Custom UI and Notifications ([#1814](https://github.com/MetaMask/snaps/pull/1814))
```
Comment (Mrtenz) [inline packages/snaps-ui/CHANGELOG.md:11]:
```suggestion
- Add support for links in custom UI and notifications ([#1814](https://github.com/MetaMask/snaps/pull/1814))
```

File: packages/snaps-utils/CHANGELOG.md [hunk 7]
```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.2.0]
+### Added
+- Add support for links in Custom UI and Notifications ([#1814](https://github.com/MetaMask/snaps/pull/1814))
```
Comment (Mrtenz) [inline packages/snaps-utils/CHANGELOG.md:11]:
```suggestion
- Add support for links in custom UI and notifications ([#1814](https://github.com/MetaMask/snaps/pull/1814))
```

File: packages/test-snaps/CHANGELOG.md [hunk 8]
```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.1.0]
+### Added
+- Add Get File Snap to test-snaps page ([#1893](https://github.com/MetaMask/snaps/pull/1893))
```
Comment (Mrtenz) [inline packages/test-snaps/CHANGELOG.md:11]:
```suggestion
- Add `snap_getFile` example snap ([#1893](https://github.com/MetaMask/snaps/pull/1893))
```

**Review by Mrtenz — CHANGES_REQUESTED** [review]

File: packages/snaps-controllers/CHANGELOG.md (hunk 1 above)
Comment (FrederikBolding) [reply packages/snaps-controllers/CHANGELOG.md:14]: How about "Fix an issue where snaps throwing a SnapError would be allowed to run longer than expected"?

**Review by Mrtenz — APPROVED** [review]

**Pull request merged**